ui/components/multichain/network-list-item-menu/network-list-item-menu.js

```diff
@@ -0,0 +1,152 @@
+import React, { useCallback, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  IconName,
+  ModalFocus,
+  Popover,
+  PopoverPosition,
+  PopoverRole,
+  Text,
+} from '../../component-library';
+import { MenuItem } from '../../ui/menu';
+import { IconColor, TextColor } from '../../../helpers/constants/design-system';
+
+export const NetworkListItemMenu = ({
+  anchorElement,
+  onClose,
+  onEditClick,
+  onDeleteClick,
+  isOpen,
+}) => {
+  const t = useI18nContext();
+
+  // Handle Tab key press for accessibility inside the popover and will close the popover on the last MenuItem
+  const lastItemRef = useRef(null);
+  const accountDetailsItemRef = useRef(null);
+  const removeAccountItemRef = useRef(null);
+  const removeJWTItemRef = useRef(null);
+
+  // Checks the MenuItems from the bottom to top to set lastItemRef on the last MenuItem that is not disabled
+  useEffect(() => {
+    if (removeJWTItemRef.current) {
```

Comment:
I don't think `removeJWTItemRef` is used -- we'll want to make sure these are used.